### ISSUE TITLE
Bug: og:image meta tag on faustjs.org not working

### ISSUE THREAD

**Issue by kellenmace** [wpengine/faustjs#983]
### Broken og:image Link

I just shared a link to faustjs.org on Twitter and the image didn't pull through:
<img width="526" alt="Screen Shot 2022-08-24 at 11 48 09 AM" src="https://user-images.githubusercontent.com/5306336/186463657-be171b31-c785-4a04-a108-60b71db8dc33.png">

According to [this blog post](https://veonr.com/blog/relative-vs-absolute-og-image-video-urls), relative URLs don't work for Open Graph images. So I think we can fix this by replacing [this code](https://github.com/wpengine/faustjs/blob/canary/internal/website/src/pages/index.js#L53-L56) with the following so that it has an absolute URL.

```jsx
        <meta
          property="og:image"
          content="https://faustjs.org/docs/img/logo-with-background.jpg"
        />
```

### Image Update

I didn't submit a PR since I think in addition to fixing this broken link, we should also swap out `/internal/website/static/docs/img/logo-with-background.jpg` with another image reflective of Faust, or eventually FaustNX.

## Applicable Versions

N/A

## Steps To Reproduce

Share a link to https://faustjs.org on Twitter or test it with https://cards-dev.twitter.com/validator to confirm that images aren't pulling through.

## The current behavior

Mega tag Images don't display on social media posts.

## The expected behavior

Mega tag Images do display on social media posts.

**Comment by josephfusco** [comment]
Hey @kellenmace thanks for reporting this! We are now tracking this with https://wpengine.atlassian.net/browse/MERL-456

**Comment by josephfusco** [comment]
Fixed in #1173
